### ISSUE TITLE
Add subcommand to fetch database

### ISSUE THREAD

**Issue by FlyingRatBull** [RustSec/cargo-audit#286]
When using `cargo audit` on multiple projects in a row it would be very useful to be able to fetch the current security database once (e.g. with `cargo audit fetch`) and use `cargo audit -n` from there on.

**Comment by tarcieri** [comment]
We can consider this, but it also seems like this problem can be solved by omitting `-n` from the first request and using it on the subsequent ones

**Comment by FlyingRatBull** [comment]
Yes that would work, but it would be more user friendly and more intuitive in my opinion if it were an own sub command. Especially as `cargo audit` essentially has two main functionalities: getting the current advisory database and checking an project against it.
